.rubocop.yml

```diff
@@ -1,9 +1,13 @@
 AllCops:
   Exclude:
     - 'lib/winrm/winrm_service.rb'
+    - 'lib/winrm/http/transport.rb'
 
 Style/Encoding:
   Enabled: true
 
 Metrics/LineLength:
   Max: 100
+
+StringLiterals:
+  Enabled: false
```

Comment:
Personally I think Rubocop's insistence for using single quotes is a bit over-prescriptive but if anyone has any strong feelings I'm willing to discuss. Personally, I'm a bit more in this camp: http://viget.com/extend/just-use-double-quoted-ruby-strings